src/Components/CreateImageWizard/validators.ts

```diff
@@ -140,8 +140,7 @@ export const isServiceValid = (service: string) => {
   // Restraints taken from service name syntax reference
   // https://www.rfc-editor.org/rfc/rfc6335#section-5.1
   return (
-    service.length <= 15 &&
-    /^[a-zA-Z0-9]([a-zA-Z0-9-]*[a-zA-Z0-9])?$/.test(service) &&
+    /^[a-zA-Z0-9]([a-zA-Z0-9-.]*[a-zA-Z0-9])?$/.test(service) &&
```

Comment:
there is any limit for the length?

Reply:
There should be a limit of 15 chars according to the linked doc, but custom first boot script's name was over a limit causing problems.

I've added the limit back and made the custom first boot script's name an exception.